test/test_rbconfig.rb

```diff
@@ -51,4 +51,21 @@ def test_vendorarchdirs
       assert_match(/\$\(sitearch|\$\(rubysitearchprefix\)/, val, "#{key} #{bug7823}")
     end
   end
+
+  def test_limits_and_sizeof_access_in_ractor
+    if defined?(Ractor)
+      assert_separately(["-W0"], <<~'RUBY')
+        r = Ractor.new do
+          sizeof_int = RbConfig::SIZEOF["int"]
+          fixnum_max = RbConfig::LIMITS["FIXNUM_MAX"]
+          [sizeof_int, fixnum_max]
+        end
+
+        sizeof_int, fixnum_max = r.take
+
+        assert_kind_of Numeric, sizeof_int, "RbConfig::SIZEOF['int'] should be a Numeric"
+        assert_kind_of Numeric, fixnum_max, "RbConfig::LIMITS['FIXNUM_MAX'] should be a Numeric"
```

Comment:
It makes sense to check they are Integer not just Numeric

Reply:
Done!